frontend/sumo-championship/src/api/api.ts

```diff
@@ -0,0 +1,102 @@
+const DOMAIN = 'http://localhost:8080/api/';
```

Comment:
HMm I don't agree with you. Because of the fact that now we have generic function for fetching data from server, we won't repeat code, and we can easily add authentication headers logic and etc. Also, I'm not usre if it is a really a problem since event if we don't use, we can't control what server will return to us

Reply:
Ye they reduce some code, but I would still wrap them around some function in some use case, as it is nice to know what are we expecting certain function to return